src/explore-education-statistics-frontend/src/middleware/pages/__tests__/rewriteDataCatalogue.test.ts

```diff
@@ -0,0 +1,27 @@
+import { NextResponse } from 'next/server';
+import rewriteDataCatalogueDownload from '../rewriteDataCatalogue';
+import middlewareTestRun from './util/middlewareTetsRun';
+
+describe('rewriteDataCatalogue', () => {
+  const redirectSpy = jest.spyOn(NextResponse, 'redirect');
+  const nextSpy = jest.spyOn(NextResponse, 'next');
+
+  test('redirects data-catalogue csv downloads to file download endpoint', async () => {
```

Comment:
A reminder that this test needs to be updated appropriately in the context of the proposed `rewrites` middleware changes:

- naming of `describe`
- naming of test case - 'rewrite' not 'redirect'